### PR TITLE
1382 - Button Group Classes updated

### PR DESCRIPTION
**Pull request by OBraunEPAM**
Regarding updating documentation of Button Group (#1382) some of the relevant classes were refactored which includes:
- deleting ButtonGroup.java class from jdi-light-bootstrap package as redundant;
- moving some of Button Group classes to a relevant packages in order to make their location met their nature;
- some of Button Group classes were renamed for the same reason as above;
- minor Code Style changes;
- few locators have been updated because they didn't work;

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttonGroup/ToolbarTests.java  2
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttonGroup/BasicTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4589854&bid=15411744)

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java [hunk 1]
```diff
@@ -27,17 +27,14 @@
...
 import io.github.com.sections.Jumbotron;
 import io.github.com.sections.MediaObjectNesting;
 import io.github.com.sections.MediaObjectSample;
 import io.github.com.sections.Toast;
-import io.github.com.sections.buttongroup.ButtonGroupBasicExample;
-import io.github.com.sections.buttongroup.ButtonGroupNesting;
-import io.github.com.sections.buttongroup.ButtonGroupSizing;
-import io.github.com.sections.buttongroup.ButtonGroupVerticalVariation;
-import io.github.com.sections.buttongroup.ButtonWithDropdown;
+import io.github.com.sections.buttongroup.*;
```
Comment (codacy-review) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java]: Codacy Issue found: [Avoid unused imports such as 'io.github.com.sections.buttongroup'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4589854)

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/inputgroup/InputGroupButtonWithDropdownTests.java [hunk 2]
```diff
@@ -1,11 +1,11 @@
-package io.github.epam.bootstrap.tests.composite.section.inputgroup;
+package io.github.epam.bootstrap.tests.composite.section.inputGroup;
```
Comment (codacy-review) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/inputgroup/InputGroupButtonWithDropdownTests.java]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4589854)

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/inputgroup/InputGroupCustomSelectWithButton.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package io.github.epam.bootstrap.tests.composite.section.inputgroup;
+package io.github.epam.bootstrap.tests.composite.section.inputGroup;
```
Comment (codacy-review) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/inputgroup/InputGroupCustomSelectWithButton.java]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4589854)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttonGroup/BasicTests.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttonGroup/ToolbarTests.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4589854&bid=15411744)

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttonGroup/ToolbarTests.java [hunk 4]
```diff
@@ -63,10 +63,10 @@ public void buttonsInButtonToolbarTest() {
...
-        buttonToolbarWithInputField.inputAreaInToolbar.is().core().attr("placeholder", placeholderForInputField);
-        buttonToolbarWithInputField.inputAreaInToolbar.setValue(textForTestingInputField);
-        assertEquals(buttonToolbarWithInputField.inputAreaInToolbar.getValue(), textForTestingInputField);
+        buttonGroupToolbarWithInputField.inputAreaInToolbar.is().displayed();
+        buttonGroupToolbarWithInputField.inputAreaInToolbar.is().enabled();
+        buttonGroupToolbarWithInputField.inputAreaInToolbar.is().core().attr("placeholder", placeholderForInputField);
+        buttonGroupToolbarWithInputField.inputAreaInToolbar.setValue(textForTestingInputField);
+        assertEquals(buttonGroupToolbarWithInputField.inputAreaInToolbar.getValue(), textForTestingInputField);
     }
 }
```
Comment (splekhanov) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttonGroup/ToolbarTests.java]: Please add new line at the end of the file

Comment (OBraunEPAM) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttonGroup/ToolbarTests.java]: done.

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttonGroup/ToolbarTests.java  2
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttonGroup/BasicTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4589854&bid=15411744)

**Pull request merged**